data/yam/agama/auto/lib/dasd.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+  dasd():: {
+    devices: [
+      {
+        channel: "0.0.0150",
+        format: true,
+        diag: true,
```

Comment:
You have to be sure the device support diagnose otherwise it will not be set online:

From https://www.ibm.com/docs/en/linux-on-systems?topic=dasds-diag-access-method

If the DIAG access method is not available and you set the use_diag attribute to 1, you cannot set the device online.

Reply:
Thanks for your info, I remove the `diag: true`, and it works fine now.